src/components/organisms/RegisterForm/RegisterForm.tsx

```diff
@@ -124,6 +130,19 @@ export const RegisterForm: React.FC<{
     [fields]
   );
 
+  const getFieldTranslate = useCallback(
+    (extra: Record<string, any>[]) => {
```

Comment:
don't use any

Reply:
this function is not needed 

the transaltion shouyld be handled by `t()` function not by any other logic 

if there are additional ytransaltion in the additional fields they should be added once you fetch additional fields with https://www.i18next.com/how-to/add-or-load-translations#add-after-init

otherwise you 
* mix component with i18n logic 
* you add secondary translation logic 
* none of above should be inside any of the components